drivers/usbc/tcpc/Kconfig

```diff
@@ -0,0 +1,28 @@
+# USBC TCPC configuration options
+
+# Copyright 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+config USBC_TCPC
```

Comment:
Suggest `menuconfig USBC_TCPC` or `menuconfig USBC` somewhere, also it is better to add _DRIVER, `menuconfig USBC_DRIVER`, if there will be an USBC stack.

Reply:
This was my request to remove the `_DRIVER` which was initially there (for the reason we generally don't add _DRIVER). But if that's a practice used for potential disambiguation that's fine with me

Reply:
> But if that's a practice used for potential disambiguation that's fine with me

I think there is a danger of confusion here, that is why we have for example Kconfig options `USB_DEVICE_DRIVER` and `USB_DEVICE_STACK`.

Reply:
I changed it to menuconfig USBC_TCPC_DRIVER

Reply:
> I changed it to menuconfig USBC_TCPC_DRIVER

Actually not, did you push a wrong state?